### PR TITLE
R4 Observation Charted Read

### PR DESCRIPTION
**Pull request by lalitha94**
Description
----
Added reads documentation
Before: 
image

After(on Local Machine):
image

PR Checklist
----
- [X] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

File: content/millennium/r4/diagnostic/observation.md [hunk 1]
```diff
@@ -119,6 +119,35 @@ Notes:
...
+
+List an individual Observation by its id:
+
+    GET /Observation/M-id
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: false, system: true) %>
+
+Note: Social History is not supported
```
Comment (lalitha94) [inline content/millennium/r4/diagnostic/observation.md:132]: Any suggestions on where to put this @joeratt

File: content/millennium/r4/diagnostic/observation.md [hunk 2]
```diff
@@ -119,6 +119,35 @@ Notes:
 
 The common [errors] and [OperationOutcomes] may be returned.
 
+## Retrieve by id
+
+List an individual Observation by its id:
+
+    GET /Observation/M-id
```
Comment (BS062721) [inline content/millennium/r4/diagnostic/observation.md:126]: what about charted blood pressures?

Comment (lalitha94) [reply content/millennium/r4/diagnostic/observation.md:126]: I have already asked Fran about this, she told told that its enough if we just include measurements as BP will be implied.

File: content/millennium/r4/diagnostic/observation.md (hunk 1 above)
Comment (joeratt) [reply content/millennium/r4/diagnostic/observation.md:132]: In the Search notes, we call them `Social history Observations`.  We may also clarify that they'll always return a 404 until later implemented?

Comment (lalitha94) [reply content/millennium/r4/diagnostic/observation.md]: Changed it https://github.com/cerner/fhir.cerner.com/commit/24672c363d3fb0c41210b31b55d272dc41322a4e#diff-d42a2575832583fdd3d20dff4a54f294R134

File: lib/resources/example_json/r4_examples_observation.rb [hunk 3]
```diff
@@ -2,6 +2,177 @@
 
 module Cerner
   module Resources
+    # TODO: All; Missing mappings
```
Comment (amcginn) [inline lib/resources/example_json/r4_examples_observation.rb:5]: TODO?

File: content/millennium/r4/diagnostic/observation.md [hunk 4]
```diff
@@ -119,6 +119,37 @@ Notes:
...
```
Comment (amcginn) [inline content/millennium/r4/diagnostic/observation.md:126]: shouldn't this just be:
    GET /Observation/:id

Comment (lalitha94) [reply content/millennium/r4/diagnostic/observation.md:126]: For observations we will have the prefix

File: lib/resources/example_json/r4_examples_observation.rb (hunk 3 above)
Comment (lalitha94) [reply lib/resources/example_json/r4_examples_observation.rb]: Thanks for pointing it out, I think it got added during the rebase.
Removed it https://github.com/cerner/fhir.cerner.com/pull/442/commits/6223f7e858de14670d4a1bb7ff11e7d6e2ca67af#diff-cd059d38e44f892deed0ab35bef7831cL5

**Comment by lalitha94** [comment]
Updated Screenshot: 
image

File: content/millennium/r4/diagnostic/observation.md (hunk 4 above)
Comment (amcginn) [reply content/millennium/r4/diagnostic/observation.md:126]: is the ID we return on a resource only the number? this indicates that we should return "M-" plus what the ID returns on the resource.

Binary is similar https://fhir.cerner.com/millennium/r4/other/binary/#retrieve-by-id

Comment (lalitha94) [reply content/millennium/r4/diagnostic/observation.md:126]: Fixed it as a part of this PR https://github.com/cerner/fhir.cerner.com/pull/445

**Comment by lalitha94** [comment]
Screenshot after the read changes are merged
ObservationReadAfterPublished

**Pull request merged**